### DISCUSSION TITLE
Crossover 23?

### DISCUSSION THREAD

**Discussion by Cytherion** [italomandara/CXPatcher#102] [Q&A] [answered]
Any plans on updating to support Crossover 23?

**Comment by italomandara** [comment]
> Any plans on updating to support Crossover 23?

We need a couple of things to happen which only Apple is in control of... our hands are tied for now

**Reply by Bentheminernz** [reply]
Is this something to do with other apps not allowed to use GPTK?

**Reply by italomandara** [reply, marked as answer]
no it's d3dmetal only working with cx22 right now they need to release a new version of the libs

**Comment by italomandara** [comment]
Full CX23.6 support is out.
